SDL/SDLMain.cpp

```diff
@@ -1183,6 +1183,7 @@ int main(int argc, char *argv[]) {
 						NativeKey(key);
 						break;
 					}
+					// TODO: Should we even keep the "non-precise" events?
```

Comment:
Probably yes, assuming we get preciseY filled in always even on Linux or elsewhere.  Not sure if any linux distros are locked to older versions of SDL2 that don't have this...

-[Unknown]